server/server.go

```diff
@@ -31,11 +31,16 @@ type Server struct {
 	doneOnce sync.Once
 }
 
+type RouterConfig struct {
+	EnableTracing bool
```

Comment:
So I think that this would be oddly breaking in a functional way (the code would work). Updating here would _disable_ this by default. I think that we should consider flipping the boolean here (e.g. `DisableTracing`).

Reply:
ooooh good point

Reply:
done in https://github.com/netlify/netlify-commons/pull/338/commits/2351a826b214c4fe13f74106833796cf81d450c6